src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/ui-data/lambda-metrics.mdx

```diff
@@ -0,0 +1,385 @@
+            <td>
+            Dead letter errors
+            </td>
+            <td>
+            Number of times the Lambda could not send an event to an async function.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            Concurrency
+            </td>
+            <td>
+            </td>
+            <td>
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Throttles
+            </td>
+            <td>
+            Number of times a function was throttled due to concurrency limits.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Concurrent executions
+            </td>
+            <td>
+            Number of concurrent executions over a given time period.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Unreserved concurrent executions
+            </td>
+            <td>
+            This shows the remaining unreserved concurrency, for unreserved functions.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            Provisioned concurrency
+            </td>
+            <td>
+            </td>
+            <td>
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Provisioned concurrent executions
+            </td>
+            <td>
+            Metric related to functions with provisioned concurrency.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Provisioned concurrency utilization
+            </td>
+            <td>
+            Percentage of provisioned concurrency being utilized at a given time.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Provisioned concurrency spillover invocations
+            </td>
+            <td>
+            Number of requests that exceeded provisioned concurrency limit and were not processed.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Provisioned concurrency reservations
+            </td>
+            <td>
+            Provisioned concurrency units reserved.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            Function
+            </td>
+            <td>
+            </td>
+            <td>
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Last modified
+            </td>
+            <td>
+            Last modified time of a function.
+            </td>
+        </tr>
+        <tr>
+            <td>
+            </td>
+            <td>
+            Cold start rate
+            </td>
+            <td>
+            Percentage of invocations that were cold started, correlation with services.
```

Comment:
@cbehera-newrelic what does "correlation with services" mean? Could you clarify that?